recipes/wasmtime/all/conanfile.py

```diff
@@ -0,0 +1,94 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration, ConanException
+import os
+
+
+class WasmtimeConan(ConanFile):
+    name = 'wasmtime'
+    homepage = 'https://github.com/bytecodealliance/wasmtime'
+    license = 'Apache-2.0'
+    url = 'https://github.com/conan-io/conan-center-index'
+    description = "Standalone JIT-style runtime for WebAssembly, using Cranelift"
+    topics = ("webassembly", "wasm", "wasi")
+    settings = "os", "compiler", "arch"
+    options = {
+        "shared": [True, False],
+        'fPIC': [True],
+    }
+    default_options = {
+        'shared': False,
+        'fPIC': True,
+    }
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15",
+            "apple-clang": "9.4",
+            "clang": "3.3",
+            "gcc": "4.9.4"
```

Comment:
Conan's version compare is pessimistic and this blocks the default settings

```suggestion
            "gcc": "4.9"
```

Reply:
Why ?
I did the same for ICC package

Reply:
The setting compare sees the patch version missing and assumes it not supported.

It was an oversight if it was missed in another PR

Reply:
Version of compiler 4.9 is not enough

Reply:
We have our lazy semver compare. You can find lots of examples in CCI

Reply:
I've already tried to run without minor version ... it is not working

Reply:
Not  working how? Can you provide a log? I'm curious about the case, because we have an official Docker image with gcc 4.9

Reply:
You can see in the last build it's being blocked

<h1 style="box-sizing: border-box; font-size: 36px; margin: 20px 0px 10px; font-family: inherit; font-weight: 500; line-height: 1.1; color: inherit;">ConanCenter - Summary of packages generated</h1><div id="summary_wrapper" class="dataTables_wrapper form-inline dt-bootstrap" style="box-sizing: border-box;"><div class="dataTables_length" id="summary_length" style="box-sizing: border-box; color: rgb(51, 51, 51); font-family: &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><label style="box-sizing: border-box; display: inline-block; max-width: 100%; margin-bottom: 5px; font-weight: normal; text-align: left; white-space: nowrap;">Show<span> </span><select name="summary_length" aria-controls="summary" class="form-control input-sm" style="box-sizing: border-box; color: rgb(85, 85, 85); font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; font-size: 12px; line-height: 30px; font-family: inherit; margin: 0px; text-transform: none; display: inline-block; width: 75px; height: 30px; padding: 5px 10px; background-color: rgb(255, 255, 255); background-image: none; border: 1px solid rgb(204, 204, 204); border-radius: 3px; box-shadow: rgba(0, 0, 0, 0.075) 0px 1px 1px inset; transition: border-color 0.15s ease-in-out 0s, box-shadow 0.15s ease-in-out 0s, -webkit-box-shadow 0.15s ease-in-out 0s; vertical-align: middle;"><option value="10" style="box-sizing: border-box;">10</option><option value="25" style="box-sizing: border-box;">25</option><option value="50" style="box-sizing: border-box;">50</option><option value="-1" style="box-sizing: border-box;">All</option></select><span> </span>entries</label></div>

status | reference | packageID | os | os_build | arch | arch_build | compiler | build_type | options | profile | build | test
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --


INVALID_CONFIGURATION | wasmtime/0.28.0 | INVALID_CONFIGURATION | Linux | Linux | x86_64 | x86_64 | gcc | 4.9 | libstdc++ |   | Release | False |   |   |  
INVALID_CONFIGURATION | wasmtime/0.28.0 | INVALID_CONFIGURATION | Linux | Linux | x86_64 | x86_64 | gcc | 4.9 | libstdc++ |   | Debug | False |   |   |  
INVALID_CONFIGURATION | wasmtime/0.28.0 | INVALID_CONFIGURATION | Linux | Linux | x86_64 | x86_64 | gcc | 4.9 | libstdc++ |   | Release | True |   |   |  
INVALID_CONFIGURATION | wasmtime/0.28.0 | INVALID_CONFIGURATION | Linux | Linux | x86_64 | x86_64 | gcc | 4.9 | libstdc++ |   | Debug | True |   |   |  

<div class="dataTables_info" id="summary_info" role="status" aria-live="polite" style="box-sizing: border-box; padding-top: 8px; white-space: nowrap; color: rgb(51, 51, 51); font-family: &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Showing 1 to 4 of 4 entries (filtered from 160 total entries)</div><div class="dataTables_paginate paging_simple_numbers" id="summary_paginate" style="box-sizing: border-box; margin: 0px; white-space: nowrap; text-align: right; color: rgb(51, 51, 51); font-family: &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><ul class="pagination" style="box-sizing: border-box; margin: 2px 0px; display: inline-block; padding-left: 0px; border-radius: 4px; white-space: nowrap;"><li class="paginate_button previous disabled" id="summary_previous" style="box-sizing: border-box; display: inline;"><br class="Apple-interchange-newline">

Reply:
Seems like this is the mostly the issue with ConanCenter ...

ConanCenter should accept all valid versions of compilers